Sources/FeaturevisorTypes/Types.swift

```diff
@@ -84,22 +84,30 @@ public enum ConditionValue: Decodable {
     case array([String])
     // @TODO: add Date type?
     // @TODO: add `null` and `undefined` somehow
-    
+
```

Comment:
@polok: I had auto-format option on in VS Code, and it ruined the formatting 😬 

could you please guide me on what command I can run to redo this as per your wishes?

the diff should be only at the very bottom.